docs/configuring-output-after.asciidoc

```diff
@@ -12,7 +12,7 @@ See <<config-sourcemapping-elasticsearch>> for more details.
 [float]
 === `fields`
 
-Fields are optional tags that can be added to any output.
+Fields are optional tags that can be added to all outputs.
```

Comment:
Since there is no ability to specify the output, I think it makes more sense so say that the are added to all outputs

```suggestion
Fields are optional tags that are added to all outputs.
```

Reply:
Looks good to me, thanks @bmorelli25